test/e2e/tests/account/account-custom-name.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { Suite } from 'mocha';
+import { Driver } from '../../webdriver/driver';
+import { withFixtures, defaultGanacheOptions } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountListPage from '../../page-objects/pages/account-list-page';
```

Comment:
```suggestion
import AccountListPage from '../../page-objects/pages/account-list-page';
import HeaderNavbar from '../../page-objects/pages/header-navbar';
```
maybe order alphabetically?

Reply:
Good suggestion, thanks !